editor/modules/aariba/behaviour.ts

```diff
@@ -21,7 +21,7 @@ let SAFE_INSERT_BEFORE_TOKENS =
 // everything except the '{' and then adding again a '\n'
 // does not behave identically. Furthermore, I think we should
 // match the brackets. (Lookup for the next possible bracket).
-let context;
+let context: any;
```

Comment:
Oh nice catch! I think I will create an issue for this one.

To understand why we have so many `any`: basically the type definition that we have for the JavaScript Ace library is incomplete. When I wrote this code, I based it on an example written in JavaScript. For many of those function I had no idea what the type was. Once, I got ideas about what it could be, I couldn't find them in the type definition. So I got lazy and didn't annotate with a type all the arguments. However, with the `noImplicitAny` check from TypeScript that I have now turned on, the use of `any` must be explicit.